docs/references/pysdk_api_reference.md

```diff
@@ -1449,7 +1449,11 @@ table_object.output(["*"]).filter("c2 = 3").to_pl()
 table_object.knn(vector_column_name, embedding_data, embedding_data_type, distance_type, topn, knn_params = None)
 ```
 
-Performs a k-nearest neighbor (KNN) or approximate nearest neighbor (ANN) vector search to identify the top k closest rows to the given vector. Suitable for working with dense vectors (dense embeddings).
+Creates a k-nearest neighbor (KNN) or approximate nearest neighbor (ANN) vector search expression to identify the top k closest rows to the given vector. Suitable for working with dense vectors (dense embeddings).
```

Comment:
Creates a dense vector search expression to identify the top n closest rows to the given sparse vector.